assets/lokomotive-kubernetes/aws/flatcar-linux/kubernetes/workers/ingress.tf

```diff
@@ -1,7 +1,7 @@
 # Target groups of instances for use with load balancers
 
 resource "aws_lb_target_group" "workers-http" {
-  name        = "${var.name}-workers-http"
+  name        = "${var.cluster_name}-${var.pool_name}-workers-http"
```

Comment:
Maybe it would make sense to remove `-workers` from it, so we save some characters? `-workers-http` is already 13 characters, + one `-`, it's 14. This leaves us only 17 characters for cluster name + pool name, not much in my opinion.